### PR TITLE
pagination and cart item

### PR DESCRIPTION
**Pull request by olena-ole**
!!copied colors from tailwind config, so there won't be any conflicts, colors are identical to those from master branch
[pagination](https://snipboard.io/tqdjK3.jpg)
<, 1, 2, 4 - default
3 - hover
5 - active/focus
'>' - disabled
when hovering over disabled - cursor forbidden
[cart item full width](https://snipboard.io/RlScDz.jpg)
'-' disabled
'+' default
[cart item small](https://snipboard.io/7oIv2a.jpg)

### CODE REVIEW

File: src/components/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number | Callback) => void;
+};
+
+const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
```
Comment (MaxSchmide) [inline src/components/Pagination.tsx:18]:
```suggestion
  perPage,
  currentPage,
  onPageChange,
```

Comment (MaxSchmide) [reply src/components/Pagination.tsx:18]: this prop isn't optional, no sense in default value

File: tailwind.config.js [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
   content: ['./src/**/*.{html,tsx}'],
   theme: {
+    colors: {
+      grey: '#e2e6e9',
+      grey_primary: '#0f0f11',
+      grey_secondary: '#89939a',
+      grey_icons: '#b4bdc3',
+      grey_background: '#fafbfc',
+      white: '#fff',
+    },
```
Comment (MaxSchmide) [inline tailwind.config.js:12]: you guys must discuss about config customization

**Review by MaxSchmide — CHANGES_REQUESTED** [review]

**Review by NazarBaraban — APPROVED** [review]

**Review by MaxSchmide — APPROVED** [review]
looks good

**Pull request merged**